datafusion/core/src/execution/context.rs

```diff
@@ -124,6 +161,7 @@ use super::options::{
 /// # #[tokio::main]
 /// # async fn main() -> Result<()> {
 /// let ctx = SessionContext::new();
+/// let df = ctx.read_csv(vec!["tests/data/example.csv", "tests/data/example.csv"], CsvReadOptions::new()).await?;
```

Comment:
I think this particular line is just confusing -- the next line ignores the result (redefines `df`) and it registers the same file twice. Given this is supposed to be the simplest example, I think we should leave this one alone.

Reply:
Makes sense. So, let's just extend `read_csv` example?